src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/ocrvalidation/client/OcrValidationClient.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.req.FormData;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.res.ValidationResponse;
+
+@Component
+public class OcrValidationClient {
+
+    private final RestTemplate restTemplate;
+
+    public OcrValidationClient(RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public ValidationResponse validate(String url, FormData formData, String s2sToken) {
```

Comment:
Could it get some s2s token provider maybe?

Reply:
my plan is for the service to take a provider in constructor

Reply:
to be more specific, I plan to create `OcrValidationService` that will take envelope input models as parameters, map them to REST api models and call the client from this PR.